packages/core-utils/src/itinerary.ts

```diff
@@ -244,6 +244,9 @@ export function getCompanyFromLeg(leg: Leg): string {
   if (from.rentalVehicle) {
     return from.rentalVehicle.network;
   }
+  if (from.vehicleRentalStation && from.vehicleRentalStation.rentalNetwork) {
```

Comment:
Try the shorter syntax if typechecking lets you do it:
```suggestion
  if (from.vehicleRentalStation?.rentalNetwork) {
```

Reply:
I'll give it a whirl!